### PR TITLE
Include feature weights in UI

### LINKED ISSUE #7: Show features and weights in UI
Show features and their weights in the UI like in the original UI.

### PR DESCRIPTION
**Pull request by JRegimbal**
Add feature weight information to the context for django and then display the names, weight value, and a bar like in the original Biollante.

### CODE REVIEW

File: biollante_rodan.py [hunk 1]
```diff
@@ -119,9 +120,10 @@ def run_my_task(self, inputs, settings, outputs):
                 temp.seek(0)
                 settings["@settings"] = temp.read()
 
-            # Preserve the number of features for certain kinds
-            # of operations the GA optimizer might perform.
+            # Preserve the number of features and weights for
+            # certain kinds of operations the GA optimizer might perform.
```
Comment (blueNoteb5) [inline biollante_rodan.py:124]: Maybe give some examples of the kinds of operations?

File: index.html [hunk 2]
```diff
@@ -7,362 +7,403 @@
     <script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
```
Comment (blueNoteb5) [inline index.html:7]: Usually the Django best practice is to put things like jQuery in a ``base.html`` template that the other templates extend, this way it's easier to be consistent across all templates if you add more in the future

Comment (blueNoteb5) [reply index.html:7]: An example: https://github.com/DDMAL/CantusDB/blob/develop/django/cantusdb_project/templates/base.html

And then this template extends it: https://github.com/DDMAL/CantusDB/blob/develop/django/cantusdb_project/templates/flatpages/default.html

File: static/index.js [hunk 3]
```diff
@@ -29,10 +29,26 @@ function handleTab (evt) {
     }
 }
 
-document.querySelectorAll(".tabs li").forEach(tab => {
+document.querySelectorAll("#settings-tabs li").forEach(tab => {
     tab.addEventListener("click", handleTab);
 });
 
+function sectionTab (evt) {
+    let li = evt.target.parentElement;
```
Comment (blueNoteb5) [inline static/index.js]: What does ``li`` mean? A more descriptive name might be better here

**Review by blueNoteb5 — COMMENTED** [review]
Everything looks good, just had a couple of nitpicks with comments and variable names and a suggestion of how to use Django templates so they're more easily extended

File: index.html (hunk 2 above)
Comment (JRegimbal) [reply index.html:7]: Yes, but this is just running as a job in rodan so any template I make here would only be accessible to this one job.

Comment (blueNoteb5) [reply index.html:7]: Oh ok, that makes sense

**Review by blueNoteb5 — APPROVED** [review]

**Pull request merged**